src/query/settings/src/lib.rs

```diff
@@ -437,6 +438,16 @@ impl Settings {
                 desc: "How many hours will the COPY file metadata expired in the metasrv, default value: 24*7=7days",
                 possible_values: None,
             },
+            SettingValue {
+                default_value: UserSettingValue::String(OPT_KEY_SNAPSHOT_LOCATION.to_string()),
+                user_setting: UserSetting::create(
+                    "hide_options_in_show_create_table",
+                    UserSettingValue::String(OPT_KEY_SNAPSHOT_LOCATION.to_string()),
+                ),
+                level: ScopeLevel::Session,
+                desc: "The list of options that needs to be ignored when executing show create table.",
```

Comment:
Sorry for the background explained:
This option is mainly for using databend in the Cloud as a SaaS(For example, https://app.databend.com), the user will never see the noise option by default.
 
So, we don't need the options as a string listing, just hide all the options is ok. 
It's better to change it to Boolean(Default is hidden).

Reply:
Understood Done.